server-spi-private/src/main/java/org/keycloak/userprofile/AttributeMetadata.java

```diff
@@ -43,6 +43,8 @@
 
     private final String attributeName;
     private String attributeDisplayName;
+    private String attributeGroup;
```

Comment:
group name is inside of the `AttributeGroupMetadata`, so separate field `attributeGroup` is not necessary here I think